client/lib/site/Makefile

```diff
@@ -2,7 +2,7 @@ NODE_BIN := $(shell npm bin)
 MOCHA ?= $(NODE_BIN)/mocha
 BASE_DIR := $(NODE_BIN)/../..
 NODE_PATH := test:$(BASE_DIR)/client:$(BASE_DIR)/shared
-COMPILERS ?= js:babel/register-without-polyfill
```

Comment:
I test this just to make sure it didn't break without the polyfill version and it was fine - 24/24 passing